README.md

```diff
@@ -23,83 +23,97 @@ v0.9.0 release, see for details: https://github.com/nutsdb/nutsdb/issues/167
-  - [Using TTL(Time To Live)](#using-ttltime-to-live)
-  - [Iterating over keys](#iterating-over-keys)
-    - [Prefix scans](#prefix-scans)
-    - [Prefix search scans](#prefix-search-scans)
-    - [Range scans](#range-scans)
-    - [Get all](#get-all)
-  - [Merge Operation](#merge-operation)
-  - [Database backup](#database-backup)
-- [Using in memory mode](#using-in-memory-mode)
-- [Using Other data structures](#using-other-data-structures)
-   - [List](#list)
-     - [RPush](#rpush)
-     - [LPush](#lpush)
-     - [LPop](#lpop)
-     - [LPeek](#lpeek)
-     - [RPop](#rpop)
-     - [RPeek](#rpeek)
-     - [LRange](#lrange)
-     - [LRem](#lrem)
-     - [LSet](#lset)
-     - [Ltrim](#ltrim)
-     - [LSize](#lsize)
-   - [Set](#set)
-     - [SAdd](#sadd)
-     - [SAreMembers](#saremembers)
-     - [SCard](#scard)
-     - [SDiffByOneBucket](#sdiffbyonebucket)
-     - [SDiffByTwoBuckets](#sdiffbytwobuckets)
-     - [SHasKey](#shaskey)
-     - [SIsMember](#sismember)
-     - [SMembers](#smembers)
-     - [SMoveByOneBucket](#smovebyonebucket)
-     - [SMoveByTwoBuckets](#smovebytwobuckets)
-     - [SPop](#spop)
-     - [SRem](#srem)
-     - [SUnionByOneBucket](#sunionbyonebucket)
-     - [SUnionByTwoBucket](#sunionbytwobuckets)
-   - [Sorted Set](#sorted-set)
-     - [ZAdd](#zadd)
-     - [ZCard](#zcard)
-     - [ZCount](#zcount)
-     - [ZGetByKey](#zgetbykey)
-     - [ZMembers](#zmembers)
-     - [ZPeekMax](#zpeekmax)
-     - [ZPeekMin](#zpeekmin)
-     - [ZPopMax](#zpopmax)
-     - [ZPopMin](#zpopmin)
-     - [ZRangeByRank](#zrangebyrank)
-     - [ZRangeByScore](#zrangebyscore)
-     - [ZRank](#zrank)
-     - [ZRevRank](#zrevrank)
-     - [ZRem](#zrem)
-     - [ZRemRangeByRank](#zremrangebyrank)
-     - [ZScore](#zscore)
-- [Comparison with other databases](#comparison-with-other-databases)
-   - [BoltDB](#boltdb)
-   - [LevelDB, RocksDB](#leveldb-rocksdb)
-   - [Badger](#badger)
-- [Benchmarks](#benchmarks)
-- [Caveats & Limitations](#caveats--limitations)
-- [Contact](#contact)
-- [Contributing](#contributing)
-- [Acknowledgements](#acknowledgements)
-- [License](#license)
+- [NutsDB ![GoDoc](https://godoc.org/github.com/xujiajun/nutsdb)  [![Go Report Card](https://goreportcard.com/badge/github.com/xujiajun/nutsdb)](https://goreportcard.com/report/github.com/xujiajun/nutsdb) <a href="https://travis-ci.org/xujiajun/nutsdb"><img src="https://travis-ci.org/xujiajun/nutsdb.svg?branch=master" alt="Build Status"></a> [![Coverage Status](https://coveralls.io/repos/github/xujiajun/nutsdb/badge.svg?branch=master)](https://coveralls.io/github/xujiajun/nutsdb?branch=master) [![License](http://img.shields.io/badge/license-Apache_2-blue.svg?style=flat-square)](https://raw.githubusercontent.com/xujiajun/nutsdb/master/LICENSE) [![Mentioned in Awesome Go](https://awesome.re/mentioned-badge.svg)](https://github.com/avelino/awesome-go#database)](#nutsdb-------)
+  - [Announcement](#announcement)
+  - [Architecture](#architecture)
```

Comment:
Personally feel that this line does not need to appear here?

Reply:
You're right. @xujiajun